src/main/java/com/provectus/collections/homeTasks/Cinderella.java

```diff
@@ -0,0 +1,37 @@
+package com.provectus.collections.homeTasks;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * @author Victor Yemelianenko vemelianenko@model.com
+ */
+
+public class Cinderella {
+    public static void main(String[] args) {
+        Random random = new Random();
+        List<Integer> listOfCinderella = new ArrayList<>();
+        for (int i = 0; i < 20; i++) {
+            listOfCinderella.add(random.nextInt(100));
+        }
+        System.out.println("Random listOfCinderella: " + listOfCinderella);
+
+        List<Integer> listDivideOnThe2 = new ArrayList<>();
+        List<Integer> listDivideOnThe3 = new ArrayList<>();
```

Comment:
according to the task, we need one more list for numbers, that does NOT divide by 2 and 3